tests/internal/test_views.py

```diff
@@ -365,8 +366,50 @@ def test_get_org_admin_bad_proxy_response(self, mock_proxy):
         mockresponse.status_code = 500
         mockresponse.json.return_value = {"error": "some error"}
         mock_proxy.get.return_value = mockresponse
-        response = self.client.get(f"/_private/api/utils/get_org_admin/123456/?type=org_id", **self.request.META)
+        response = self.client.get("/_private/api/utils/get_org_admin/123456/?type=org_id", **self.request.META)
         self.assertEqual(response.status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
         string_data = response.content.decode()
         dictionary_data = eval(string_data.replace("null", "None"))
         self.assertEqual(dictionary_data, mockresponse.json.return_value)
+
+    @override_settings(INTERNAL_DESTRUCTIVE_API_OK_UNTIL=invalid_destructive_time())
+    def test_delete_selective_roles_disallowed(self):
+        """Test that we cannot delete selective roles when disallowed."""
+        response = self.client.delete(f"/_private/api/utils/roles/{self.role.uuid}/", **self.request.META)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.content.decode(), "Destructive operations disallowed.")
+
+    @override_settings(INTERNAL_DESTRUCTIVE_API_OK_UNTIL=valid_destructive_time())
+    def test_delete_selective_roles(self):
+        """Test that we can delete selective roles when allowed and no roles."""
+        response = self.client.delete("/_private/api/utils/roles/{uuid4()}/", **self.request.META)
```

Comment:
I believe `f` is missing here `f"/_private/api/utils/roles/{uuid4()}/"` even the result (404) will be same

Reply:
Oh, good catch!